### ISSUE TITLE
Create a Modal when user clicks on Logout link with Yes or No

### ISSUE THREAD

**Issue by Rajdeepc** [Rajdeepc/iam-available#1]
1. Create a Modal when user clicks on Logout link with "Are you sure you want to logout?" and Yes or No buttons
    1a. Yes button in green in capital letters
    1b. No button in grey in capital letters.
2. If user clicks on Yes -> call logout()
3. If user clicks on No --> Only dismiss the modal
4. Create a PR to master branch

**Comment by wojciechwachowiak** [comment]
Hi, can you assign me?

**Comment by Natalia504** [comment]
I can take it! But, in a demo app, how do I login in order to logout? I get 404. Is there a tes-user?

**Comment by Rajdeepc** [comment]
Can you rephrase your question? 
You need to start your local mongodb
once you run this app you will be asked to signup
after you will see logout link

**Comment by Natalia504** [comment]
Oh I see! I haven't run it locally yet. Just looked at the demo and didn't find the Register button. Are you ok with using some kind of a UI Library like MaterialUI, Antd, or ReactBootstrap?

**Comment by Rajdeepc** [comment]
No pls dont use any other library. create a custom modal component. it will be a small component

**Comment by shashwatbangar** [comment]
hello @Rajdeepc  can you please assign me this project I would love to work on it. I understood what you want. I will be only using HTML CSS Node.js for this. Can you please assign me?

**Comment by Rajdeepc** [comment]
@shashwatbangar I have already assigned to Natalia. She has already started working on it. Please have a look at my other projects. There are few open issues to work on.

**Comment by Rajdeepc** [comment]
Issue Solved
